### DISCUSSION TITLE
Beginners Questions

### DISCUSSION THREAD

**Discussion by fxmb** [mirumee/saleor-dashboard#2830] [General] [answered]
I find the docs very confusing, there is multiple repos, everything seems a bit cluttered and hard to find...
A) how can I upload my products to the store (we have around 200k products)
B) how can I adjust the product schema in dashboard? Would I do that directly via the django code?
C) how can I login locally? I dont have an account / PW yet

**Comment by timuric** [comment, marked as answer]
Saleor is a headless platform, meaning there are core API and the rest is adjecent to that, therefore each project has different repository. For core docs you can all find everything here
https://docs.saleor.io/docs/3.x/

We are also working on a new documentation structure with more indepth guides. Stay tuned for updates https://saleor.typeform.com/to/JTJK0Nou

> A) how can I upload my products to the store (we have around 200k products)

You can either run graphql queries locally or write SQL, we discourage the later since it can yield unexpecteed errors later

> B) how can I adjust the product schema in dashboard? Would I do that directly via the django code?

Check out settings, product types, this is where you can create various product attribute models

> C) how can I login locally? I dont have an account / PW yet

You would need to run dashboard and backend https://github.com/saleor/saleor-platform

Alternatively we recommend using saleor cloud, which provides you with SaaS like experience while being more cost effective than poopular hosting options.
